blocks/editable/format-toolbar/index.js

```diff
@@ -168,12 +172,14 @@ class FormatToolbar extends Component {
 		);
 
 		return (
-			<div className="blocks-format-toolbar">
+			<div className="blocks-format-toolbar" >
```

Comment:
Why was this change made? By convention we tend to omit the space before the opening. Could probably be enforced with a rule in the `eslint-plugin-react` set.

Reply:
Accidental.